src/pages/apps/[systemId]/[appId]/launch.js

```diff
@@ -53,6 +56,25 @@ export default () => {
         },
     });
 
+    const [submitAnalysisMutation] = useMutation(
+        ({ submission }) => submitAnalysis(submission),
+        {
+            onSuccess: (resp, { onSuccess }) => {
+                // return a promise so that mutate only resolves after this callback
+                return queryCache
+                    .refetchQueries(getAnalysesQueryKey)
+                    .then(() => {
+                        router.push(`/${NavigationConstants.ANALYSES}`);
+                        onSuccess(resp);
+                    });
```

Comment:
Do we have refetch analyses here ? Won't get get fetched by default when analyses page is loaded ?

Reply:
I'm not that familiar with `react-query` yet, so I don't really know. I tried asking in a previous comment (https://github.com/cyverse-de/sonora/pull/121#discussion_r439698956) but didn't get a specific answer.

I can try removing the refetch and see what happens 🤷‍♂️

Reply:
Oops. Sorry :) My guess and based on my understanding this `refetch` call is not needed. Lets see how wrong I am 😜

Reply:
Did this work ?

Reply:
Yep, looks like it works 👍